aep/general/0202/aep.md.j2

```diff
@@ -0,0 +1,52 @@
+# Fields
+
+The [`buf.validate.field`][protovalidate] type, through its accompanying
+extension `buf.validate.field`, enriches a field's schema beyond the basic
+name and type information.
+
+## Guidance
+
+Decorating a field with `buf.validate.field` is only necessary when a field
```

Comment:
```suggestion
Annotating a field with `buf.validate.field` is only necessary when a field
```

I think this is a more common term, and we should probably try to be consistent about it.